test/csv/write/test_general.rb

```diff
@@ -205,6 +205,12 @@ def test_encoding_euc_jp
     assert_equal(%Q[あ,い,う#{$INPUT_RECORD_SEPARATOR}].encode("EUC-JP"),
                  generate_line(row))
   end
+
+  def test_write_converters
+    assert_equal(%Q[=a,=b,=c\n],
+                 generate_line(["a", "b", "c"],
+                               write_converters: ->(value) { value.prepend('=') } ))
```

Comment:
Could you avoid using destructive method in converter`?
Converter should return converted value.

Reply:
Another one I was a bit confused. I think I fixed it but it would be nice if you can take a look at it again.

Reply:
`value.prepend('=')` is a destructive method call.
`'=' + value` isn't a destructive method call.

```ruby
a = "x"
"=" + a
p(a) # => "x"
a.prepend("=")
p(a) # => "=x"
```